src/main/scala/fr/acinq/bitcoin/scalacompat/package.scala

```diff
@@ -83,22 +83,16 @@ package object scalacompat {
    * @param script    public key script
    * @return the address of this public key script on this chain
    */
-  def computeScriptAddress(chainHash: BlockHash, script: Seq[ScriptElt]): Either[AddressFromPublicKeyScriptResult.Failure, String] = addressFromPublicKeyScript(chainHash, script)
+  def computeScriptAddress(chainHash: BlockHash, script: Seq[ScriptElt]): Either[fr.acinq.bitcoin.BitcoinError, String] = addressFromPublicKeyScript(chainHash, script)
```

Comment:
```suggestion
def computeScriptAddress(chainHash: BlockHash, script: Seq[ScriptElt]): Either[BitcoinError, String] = addressFromPublicKeyScript(chainHash, script)
```

Reply:
Done in https://github.com/ACINQ/bitcoin-lib/pull/85/commits/6e6cf9e17a8927a20f1afc075fd8687e264d0924